src/app/js/fields/sourceValue/SourceValueToggle.js

```diff
@@ -40,6 +42,20 @@ const TRANSFORMERS_FORM_STATUS = new Map([
             },
         ],
     ],
+    [
+        'arbitrary',
```

Comment:
@AlasDiablo why why not use the keyword “routine”, that would certainly be more meaningful.

Reply:
I have made this change for backwards compatibility, but upon further testing, it is not used in the models and can be renamed.